src/main/java/com/online/store/service/ProductServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package com.online.store.service;
+
+import com.online.store.dao.ProductDao;
+import com.online.store.lib.Inject;
+import com.online.store.lib.Service;
+import com.online.store.model.Product;
+import java.util.List;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+    @Inject
+    ProductDao productDao;
+
+    @Override
+    public Product create(Product product) {
+        productDao.create(product);
+        return product;
```

Comment:
create() returns Product obj, so here you can write it in one line too

Reply:
agree)